recipes/emg3d/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "emg3d" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0a027ccae7f13cbbde903aab9fbedb89da97c72af90ddb566ddec09c0a5f8311
+
+build:
+  skip: True    # [win]  # Remove when numba-feedstock/pull/37 is merged.
+  # noarch: python       # Enable when numba-feedstock/pull/37 is merged.
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  preserve_egg_dir: True
+
+requirements:
+  host:
+    - empymod
+    - numba >=0.40.0
+    - numpy >=1.15.0
+    - pip
+    - python >=3.7
+    - setuptools_scm
+    - scipy >=1.1.0
+  run:
+    - empymod
+    - numba >=0.40.0
+    - numpy >=1.15.0
```

Comment:
See this info for how to properly specify a numpy constraint in conda-forge: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
@beckermr, that requirement is for packages using the numpy C API

Reply:
So should I change it back to how it was originally?

Reply:
Ah good point @isuruf! I am not sure it needs be change but feel free in the feedstock.

Reply:
It should be like,

```
requirements:
  host:
    - pip
    - python >=3.7
    - setuptools_scm
  run:
    - empymod
    - numba >=0.40.0
    - numpy >=1.15.0
    - python >=3.7
    - scipy >=1.1.0
```

It's better to change, so that you don't have to change the numpy requirement in host in the future.

Reply:
OK, will do. I see you don't put `numba`, `numpy`, and `scipy` in the host-section, so I'll take that out for both feedstocks of `empymod` and `emg3d`.

Thanks both°